config/services/mailer/sender.xml

```diff
@@ -11,13 +11,18 @@
 
 -->
 
-<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+<container
+    xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
     <services>
-        <service id="sylius_mollie.email_sender.payment_link" class="Sylius\MolliePlugin\Mailer\Sender\PaymentLinkEmailSender">
+        <defaults public="true" />
+
+        <service id="sylius_mollie.mailer.sender.payment_link" class="Sylius\MolliePlugin\Mailer\Sender\PaymentLinkEmailSender">
```

Comment:
Sometimes we use `payment_link` and sometimes `paymentlink`, could we unify it? 🤔

Reply:
Went with the former globally :wave: